tools/dbus_ignorelist

```diff
@@ -558,6 +558,9 @@ org.freedesktop.systemd1.Manager.ConfirmSpawn
 org.freedesktop.systemd1.Manager.ControlGroup
 org.freedesktop.systemd1.Manager.CtrlAltDelBurstAction
 org.freedesktop.systemd1.Manager.DefaultBlockIOAccounting
+org.freedesktop.systemd1.Manager.DefaultCoredumpLimitInterval
```

Comment:
don't add these here, instead list them at the bottom of the manpage with the version they were added in